secretkeys/secretkeys.go

```diff
@@ -0,0 +1,39 @@
+package secretkeys
+
+import (
+	"strings"
+
+	"github.com/bitrise-io/go-utils/v2/env"
+)
+
+const (
+	// EnvKey is the shared env var key
+	EnvKey    = "BITRISE_SECRET_ENV_KEY_LIST"
+	separator = ","
+)
+
+// Manager ...
```

Comment:
Nit: we no longer need this dummy comments, the master branch now uses `golangci-lint` instead of `go-lint` that was enforcing these comments.

Reply:
Nice, I will remove these.

Reply:
So it turns out this repo is not hooked into the unified CI so I need comments for now. I will upgrade this to the unified CI once my ticket is done.